core/domain/question_domain.py

```diff
@@ -229,3 +230,28 @@ def update_question_data(self, question_data):
             question_data: dict. A dict representing the question data.
         """
         self.question_data = question_data
+
+    def get_skills(self):
+        """Fetches the skills associated with the question."""
+        collection = collection_services.get_collection_by_id(
+            self.collection_id)
+        skills = collection.skills
+
+        question_skills = []
+        for skill in skills:
+            if self.question_id in skill.question_ids:
+                question_skills.append(skill)
+        return question_skills
+
+    def can_user_answer_question(self, user_id):
+        """Verifies if the current user has the acquired
+        skills to answer the question.
+
+        Returns:
+            A boolean representing True or False.
+        """
+        question_skills = self.get_skills()
+        user_skills = collection_services.get_acquired_skills_of_user(user_id)
+        #Returns True if questions_skills are present in user_skills
```

Comment:
This comment isn't needed, I think. It just repeats what the line below is doing.

Reply:
Updated.